deploy/ansible/deploy.yml

```diff
@@ -30,37 +30,11 @@
       include: includes/topology_add_node.yml
       loop: "{{topology}}"
 
-    # If we are installing to localhost, and are not using a packaged version
-    # we need to special case where configuration files come from.
-    - name: Set configuration file location
-      set_fact:
-        "template_path": "/srv/shakenfist/venv/share/shakenfist/templates"
```

Comment:
`template_path` appears to still be used, without it `TASK [Copy libvirt template]` fails with
```
ansible.vars.hostvars.HostVarsVars object' has no attribute 'template_path'
```

Reply:
Nice catch, thanks.